internal/config/config_test.go

```diff
@@ -293,6 +340,7 @@ func TestGetConfig(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.description, func(t *testing.T) {
+			defer setGetLdConfigPathForTest()
```

Comment:
Question: why do we `defer` the execution of this function call? Don't we need to set the ldconfig path for the test case we are about to execute?

Reply:
You're right. This should be:
```suggestion
			defer setGetLdConfigPathForTest()()
```